Neos.ContentRepositoryRegistry/Classes/Service/EventMigrationService.php

```diff
@@ -498,31 +499,14 @@ public function migrateWorkspaceMetadataToWorkspaceService(\Closure $outputFn):
 
             switch ($eventType) {
                 case 'RootWorkspaceWasCreated':
-                    $eventData = self::decodeEventPayload($eventEnvelope);
-                    if (!isset($eventData['workspaceTitle'])) {
-                        // without the field it's not a legacy workspace creation event
-                        continue 2;
-                    }
-                    $workspaces[$eventData['workspaceName']] = [
-                        'workspaceName' => $eventData['workspaceName'],
-                        'workspaceTitle' => $eventData['workspaceTitle'],
-                        'workspaceDescription' => $eventData['workspaceDescription'],
-                        'baseWorkspaceName' => null,
-                        'workspaceOwner' => null
-                    ];
-                    break;
                 case 'WorkspaceWasCreated':
                     $eventData = self::decodeEventPayload($eventEnvelope);
-                    if (!isset($eventData['workspaceTitle'])) {
-                        // without the field it's not a legacy workspace creation event
-                        continue 2;
-                    }
```

Comment:
without this statement

> 'Found %d legacy workspace events resulting in %d workspaces.

is wrong. .. also this is okay not to be there? We have runtime migration fallback? Why did we remove this check?